### PR TITLE
[Automation][Cephadm]Add osd crush device class scenario

### PR DESCRIPTION
**Pull request by MohitBis**
Scenario: Verify osd spec class crush_device_class

### CODE REVIEW

**Comment by MohitBis** [comment]
Logs:

Quincy- http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-K6OIRI/
Reef- http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-62361Y/

File: tests/cephadm/test_osd_crush_device_class.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    """Verify osd spec class crush_device_class"""
+    # Get config
+    config = kwargs.get("config")
+
+    # Check mandatory parameter crush_device_class
+    if not config.get("crush_device_class"):
+        raise ConfigError("Mandatory config 'crush_device_class' not provided")
+
+    # get cluster config
+    installer = installer = ceph_cluster.get_nodes(role="installer")[0]
```
Comment (tintumathew10) [inline tests/cephadm/test_osd_crush_device_class.py:16]: any reason why multiple assignment to same variable ? 
installer = installer = ceph_cluster.get_nodes(role="installer")[0]

Comment (MohitBis) [reply tests/cephadm/test_osd_crush_device_class.py]: Thanks @tintumathew10 

Updated

**Review by tintumathew10 — COMMENTED** [review]
LGTM

File: tests/cephadm/test_osd_crush_device_class.py (hunk 1 above)
Comment (tintumathew10) [reply tests/cephadm/test_osd_crush_device_class.py]: Thanks Mohit

**Review by vamahaja — APPROVED** [review]

**Pull request merged**